nbtlib/tag.py

```diff
@@ -387,6 +387,9 @@ def parse(cls, buff, byteorder='big'):
     def write(self, buff, byteorder='big'):
         write_string(self, buff, byteorder)
 
+    def __str__(self):
+        return str.__str__(self)
+

```

Comment:
If you really want to do that,
it seems that you can simply write 
`__str__ = str.__str__`

Reply:
True, I agree `__str__ = str.__str__` is a better implementation